bears/cmake/CMakeLintBear.py

```diff
@@ -21,11 +23,19 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         max_line_length: int=79,
                          cmakelint_config: path=''):
         """
+        :param max_line_length:
+             Maximum number of characters for a line.
++            When set to 0 allows infinite line length.
         :param cmakelint_config: The location of the cmakelintrc config file.
         """
+        if not max_line_length:
+            max_line_length = sys.maxsize
+
         args = ()
         if cmakelint_config:
-            args += ('--config=' + cmakelint_config,)
+            args += ('--config=' + cmakelint_config,
+                     '--linelength=' + str(max_line_length),)
```

Comment:
your logic isnt working yet, because you have not tried to *prove* that this feature is actually working correctly.

Reply:
still not working.

Reply:
@kriti21 have you checked that the argument to be given for this is `linelength` . Just asking coz its not working.

Reply:
`linelength` is a valid argument. https://github.com/richq/cmake-lint/#14

Reply:
read your code changes.
carefully.

Reply:
IMO this sud come outside the if condition.

Reply:
```
        if cmakelint_config:
            args += ('--config=' + cmakelint_config,)

        if max_line_length:
            args += ('--linelength=' + str(max_line_length),)
```
Why I am suggesting to break and make it in 2 conditions is because if a user does not give `cmakelint_config` then your code has no use of `max_line_length` that you have given [here](https://github.com/coala/coala-bears/pull/2364/files#diff-f5c369821ed5f2660ab333edc21ed4c7R35).

Reply:
there is no need for `if max_line_length`. Consider what happens when `max_line_length` is 0.

Reply:
@sangamcse thats what is written in the above review :P

Reply:
just do it like ` return args + (--linelength.... , filename, )`

Reply:
Then `max_line_length = sys.maxsize` @anctartica

Reply:
that condition `if not max_line_length` will get triggered as it already might have changed to sys.maxsize ;P